README.md

```diff
@@ -237,7 +197,7 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/Lasten-Ernest/AwesomeBooks-With_ES6/issues).
```

Comment:
- [ ] @Lasten-Ernest kindly check on your issues page link, its broken am getting a 404 error 
![image](https://user-images.githubusercontent.com/61408860/221110886-20b16a0c-f625-45d3-b4bd-a860c37f21d5.png)

Reply:
ReadMe and css files updated